test/suite/test_las08.py

```diff
@@ -0,0 +1,131 @@
+
```

Comment:
Why not write the test in las06?

Reply:
You can keep it as a separate test. I'm not going to argue against that. But I feel this should be in las06 as they all test the same code.

Reply:
Honestly I prefer tests to be broken and out and test as few things in one test as possible rather than one big file of many tests. I'm happy to keep it like this if you are.

Reply:
That is fine for me.